.github/CONTRIBUTING.md

```diff
@@ -1,47 +1,22 @@
 # Contributing to PowerShell
 
 We welcome and appreciate contributions from the community.
-There are many ways to become involved with PowerShell:
-including filing issues,
-joining in design conversations,
-writing and improving documentation,
-and contributing to the code.
-Please read the rest of this document to ensure a smooth contribution process.
-
-## Intro to Git and GitHub
-
-* Make sure you have a [GitHub account](https://github.com/signup/free).
-* Learning Git:
-  * GitHub Help: [Good Resources for Learning Git and GitHub][good-git-resources]
-  * [Git Basics](../docs/git/basics.md): install and getting started
-* [GitHub Flow Guide](https://guides.github.com/introduction/flow/):
-  step-by-step instructions of GitHub Flow
-
-## Quick Start Checklist
-
-* Review the [Contributor License Agreement][CLA] requirement.
-* Get familiar with the [PowerShell repository](../docs/git).
-
-## Contributing to Issues
+There are many ways to become involved with PowerShell including:
+- filing and participating in issues
+- joining in design conversations
+- writing and improving documentation
+- contributing to the code
 
-* Review [Issue Management][issue-management].
-* Check if the issue you are going to file already exists in our [GitHub issues][open-issue].
-* If you can't find your issue already,
-  [open a new issue](https://github.com/PowerShell/PowerShell/issues/new/choose),
-  making sure to follow the directions as best you can.
-* If the issue is marked as [`Up-for-Grabs`][up-for-grabs],
-  the PowerShell Maintainers are looking for help with the issue.
-* Issues marked as [`First-Time-Issue`][first-time-issue],
-  are identified as being easy and a great way to learn about this project and making
-  contributions.
+Please read the rest of this document to ensure a smooth contribution process.
 
 ## Contributing to Documentation
+Contributing to the docs is an excellent way to get started with the process of making open source contributions with minimal technical skill required.
 
-### Contributing to documentation related to PowerShell
+Please see the [Contributor Guide in `MicrosoftDocs/PowerShell-Docs`](https://aka.ms/PSDocsContributor).
 
-Please see the [Contributor Guide in `MicrosoftDocs/PowerShell-Docs`](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md).
+Learn how to [Contribute to Docs like a Microsoft Insider](https://www.youtube.com/watch?v=ZQODV8krq1Q) (by @sdwheeler)
 
-#### Quick steps if you're changing an existing cmdlet
+### Updating Documentation for an existing cmdlet
```

Comment:
If we have link to official doc contributor above maybe we could remove the steps below at all? Or move to the official docs?

/cc @sdwheeler

Reply:
@iSazonov The Docs contributor guide is about the process of contributing docs in the docs repo. The steps listed here are about ensuring your changes to code get documented in docs. That information belongs here.